Mage.Tests/src/test/java/org/mage/test/cards/single/mid/SigardasSplendorTest.java

```diff
@@ -0,0 +1,76 @@
+package org.mage.test.cards.single.mid;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ * {@link mage.cards.s.SigardasSplendor Sigarda's Splendor}
+ * {2}{W}{W}
+ * Enchantment
+ * As Sigarda’s Splendor enters the battlefield, note your life total.
+ * At the beginning of your upkeep, draw a card if your life total is greater than or equal
+ * to the last noted life total for Sigarda’s Splendor. Then note your life total.
+ * Whenever you cast a white spell, you gain 1 life.
+ *
+ * @author notgreat
+ */
+public class SigardasSplendorTest extends CardTestPlayerBase {
```

Comment:
looks like this file is indented with tabs? spaces are standard code style here

Reply:
Oops, I copied from the other `MID` test which also happened to be using tabs. Will fix both of them.